tech.json

```diff
@@ -257,7 +257,14 @@
   },
   {
     "name": "canHerdBabyTurtles",
-    "requires": [],
+    "requires": [
+      {"cost": [
```

Comment:
I feel like moving the baby turtles is separate from actually executing the shinespark, so I'd leave that damage at the actual link level.

What do you think?

Reply:
As I hardly know how it's performed I'm just gonna change back.